home-application/src/main/java/com/softserveinc/ita/homeproject/application/exception/mapper/ConstraintViolationMapper.java

```diff
@@ -0,0 +1,49 @@
+package com.softserveinc.ita.homeproject.application.exception.mapper;
+
+import com.softserveinc.ita.homeproject.homeservice.exception.BaseHomeException;
+import org.hibernate.validator.internal.engine.ConstraintViolationImpl;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.ConstraintViolationException;
+
+import javax.validation.Path;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.Provider;
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
You need to change imports order according to code style. CI fails because you have import order violations.